Light/include/light/rendering/camera.hpp

```diff
@@ -10,11 +10,12 @@ namespace Light
 	public:
 		Camera(glm::mat4 projectionMatrix = glm::mat4(1.0f))
 			: m_projectionMatrix(projectionMatrix)
-		{}
+		{
+		}
 
 		virtual ~Camera() = default;
 
-		inline const glm::mat4& getProjectionMatrix() { return m_projectionMatrix; }
+		inline const glm::mat4 &getProjectionMatrix() { return m_projectionMatrix; }
```

Comment:
Typically, we [take the `&` to be a part of the type](https://stackoverflow.com/a/49360152/12591388).